modules/jenkins-plugin/src/main/resources/com/smartbear/ready/jenkins/RemoteTestStarter/global.jelly

```diff
@@ -0,0 +1,32 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <!--
+    This Jelly script is used to produce the global configuration option.
+
+    Jenkins uses a set of tag libraries to provide uniformity in forms.
+    To determine where this tag is defined, first check the namespace URI,
+    and then look under $JENKINS/views/. For example, <f:section> is defined
+    in $JENKINS/views/lib/form/section.jelly.
+
+    It's also often useful to just check other similar scripts to see what
+    tags they use. Views are always organized according to its owner class,
+    so it should be straightforward to find them.
+  -->
+  <f:section title="ReadyApi TestServer Plugin">
+      <f:entry title="Server URL" field="serverUrl"
+               description="The URL to the root of the TestServer, e.g. https://our.testserver.instance:8443/v1.">
```

Comment:
well - i'm guessing that when/if we release a v2 version of the server - we will update the client also to use v2 automatically. The server should continue to support v1 (if possible) - so any existing clients should continue to work